### PR TITLE
Make compatible with ftw.theming 2.0.0

### PR DESCRIPTION
**Pull request by bierik**
⚠️ Requires https://github.com/4teamwork/ftw.theming/pull/85

### CODE REVIEW

**Review by mbaechtold — CHANGES_REQUESTED** [review]
Please update the required version in https://github.com/4teamwork/ftw.mobile/blob/master/setup.py#L61

**Comment by bierik** [comment]
@mbaechtold updated

File: setup.py [hunk 1]
```diff
@@ -58,7 +58,7 @@
           'Products.CMFCore',
           'Zope2',
           'ftw.gopip',
-          'ftw.theming >= 1.6.0',
+          'ftw.theming >= 2.0.0',
```
Comment (mbaechtold) [inline setup.py:61]: This has not been released yet?!?

**Review by mbaechtold — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (bierik) [reply setup.py:61]: I know thats why it requires https://github.com/4teamwork/ftw.theming/pull/85 😄

**Comment by bierik** [comment]
@mbaechtold ftw.theming 2.0.0 is released and test are passing

**Review by mbaechtold — APPROVED** [review]

**Pull request merged**